Dockerfile

```diff
@@ -45,7 +45,10 @@ RUN apt-get update && apt-get install -y \
 # to regenerate the U-Boot boot.scr on platforms that need customization
     u-boot-tools \
 # needed to run pxz
-    libgomp1
+    libgomp1  \
+# zip and unzip archive
+    zip  \
+    unzip
```

Comment:
Curious, do these have parallel alternatives? E.g  `pgzip` and `pxz` which we use for the other compression types.

Probably not

Reply:
I'm sure there is, didn't find anything right now though. Easy to fix up later if we do though